Source/WebCore/page/Quirks.cpp

```diff
@@ -1830,7 +1830,25 @@ void Quirks::setTopDocumentURLForTesting(URL&& url)
 }
 
 // FIXME(rdar://141554467): The set of static functions below will be generated from a JSON file in a future patch. For now, we just move the logic
-// for deciding if a particular quirk is needed to domain-specific functionss below:
+// for deciding if a particular quirk is needed to domain-specific functions below:
+#if PLATFORM(IOS) || PLATFORM(MAC)
```

Comment:
Is this intentionally `PLATFORM(IOS)`-only? (i.e. excluding watchOS, visionOS, Catalyst)